keps/sig-instrumentation/1602-structured-logging/kep.yaml

```diff
@@ -20,11 +20,10 @@ approvers:
 see-also:
 replaces:
 stage: beta
-latest-milestone: "v1.23"
+latest-milestone: "v1.35"
 milestone:
   alpha: "v1.19"
   beta: "v1.23"
-  stable: "v1.26"
```

Comment:
This was too optimistic... there's currently no projection when this will be considered sufficiently complete to declare GA.